src/file_picker.js

```diff
@@ -1,28 +1,15 @@
-import jQuery from 'jquery';
 import Vue from 'vue';
 import FilePicker from './components/file_picker.vue';
-import FileExplorer from './file_explorer';
+import {
+  file_system_favorites
```

Comment:
`file_system_favorites` is something that multiple file pickers can take advantage of, so it is pulled out into its own run-once function. Since `fetch` does not have a synchronous mode the function has been altered to return a Promise to allow everything else to wait until it is ready.